build-image/first-boot.bash

```diff
@@ -161,10 +162,11 @@ if ! cond_redirect dpkg -s "git" &>/dev/null; then
   if apt-get install --yes git &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 fi
 
-if [ -d /opt/openhabian ]; then cd /opt && rm -rf /opt/openhabian; fi
+# must not remove for offline to work
+#if [ -d /opt/openhabian ]; then cd /opt && rm -rf /opt/openhabian; fi
```

Comment:
I suppose that removing the delete will cause `git clone` command to fail....
```
fatal: destination path '/opt/openhabian already' exists and is not an empty directory.
```

Reply:
right, good you spotted that.
How about this now ?